server/src/main/resources/logback.xml

```diff
@@ -11,12 +11,33 @@
     <encoder>
       <pattern>%date %level [%thread] %logger{10} [%file:%line] %msg%n</pattern>
     </encoder>
+    <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+      <!-- daily rollover -->
+      <fileNamePattern>logs\logFile.%d{yyyy-MM-dd}.%i.html</fileNamePattern>
```

Comment:
`.html`? Curious, why html?

Reply:
ah...good catch! This was copy pasted from stackoverflow for rotating logs...